thorlcr/graph/thgraph.cpp

```diff
@@ -2729,12 +2734,12 @@ void CJobBase::startJob()
     setPerformanceMonitorHook(perfmonhook);
     PrintMemoryStatusLog();
     logDiskSpace();
-    unsigned keyNodeCacheMB = (unsigned)getWorkUnitValueInt("keyNodeCacheMB", DEFAULT_KEYNODECACHEMB * queryJobChannels());
-    unsigned keyLeafCacheMB = (unsigned)getWorkUnitValueInt("keyLeafCacheMB", DEFAULT_KEYLEAFCACHEMB * queryJobChannels());
-    unsigned keyBlobCacheMB = (unsigned)getWorkUnitValueInt("keyBlobCacheMB", DEFAULT_KEYBLOBCACHEMB * queryJobChannels());
-    setNodeCacheMem(keyNodeCacheMB * 0x100000);
-    setLeafCacheMem(keyLeafCacheMB * 0x100000);
-    setBlobCacheMem(keyBlobCacheMB * 0x100000);
+    keyNodeCacheMB = (unsigned)getWorkUnitValueInt("keyNodeCacheMB", DEFAULT_KEYNODECACHEMB * queryJobChannels()) * 0x100000;
```

Comment:
No longer stored in MB, and not unreasonable to overflow 4G on some systems, so unsigned is too small.

Reply:
agree, will fix.